### ISSUE TITLE
forkExec leaves zombies

### ISSUE THREAD

**Issue by toke** [Alexays/Waybar#369]
Processes spawned by `forkExec` - via `on-click` or `on-click-right` leaves zombie processes.

To reproduce.

```json
   "custom/demo": {
        "format": "Zombienator{}",
        "exec": "date",
        "return-type": "newline",
        "on-click": "touch /tmp/zombietest",
        "on-click-right": "rm /tmp/zombietest"
    }
```
After clicking on the "Zombienator" defunct processes are visible in the process list.

**Comment by toke** [comment]
Probably a similar issue is happening with the other `open` method.

**Comment by Alexays** [comment]
Fixed in #370
